.github/workflows/label.yml

```diff
@@ -89,3 +89,15 @@ jobs:
           - [maintenance](https://github.com/pyansys/pyDyna/pulls?q=label%3Amaintenance+)
           - [release](https://github.com/pyansys/pyDyna/pulls?q=label%3Arelease+)
           - [testing](https://github.com/pyansys/pyDyna/pulls?q=label%Atesting+)
+
+  changelog-fragment:
+      name: "Create changelog fragment"
+      needs: [commenter]
+      permissions:
+        contents: write
+        pull-requests: write
+      runs-on: ubuntu-latest
+      steps:
+      - uses: ansys/actions/doc-changelog@v8
+        with:
+          token: ${{ secrets.PYANSYS_CI_BOT_TOKEN }}
```

Comment:
```suggestion
          token: ${{ secrets.PYANSYS_CI_BOT_TOKEN }}
          bot-user: ${{ secrets.PYANSYS_CI_BOT_USERNAME }}
          bot-email: ${{ secrets.PYANSYS_CI_BOT_EMAIL }}
```

Reply:
You could also add "use-conventional-commits: true" if you want the changelog to be categorized by conventional commit types in PR titles instead of being categorized by labels